src/Microsoft.VisualBasic/tests/VB/PathFeatures.vb

```diff
@@ -0,0 +1,90 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+Option Explicit On
+Option Strict On
+
+' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Reflection
+Imports System.Runtime.InteropServices
+Namespace Microsoft.VisualBasic.Tests.VB
+    Partial Public Module PathFeatures
```

Comment:
Please remove the clone of `PathFeatures`. If the C# implementation cannot be used directly, we should simplify the caller.

Reply:
I removed it and just call the one DLL I need in Windows.

Reply:
@cston LongDirectoryPathTest is still failing on one Windows platform I am adding some tests to figure out why then I will simplify. If you know how to detect the failing platform I will remove all this detection code.